Source/WebCore/rendering/RenderReplaced.h

```diff
@@ -52,14 +52,14 @@ class RenderReplaced : public RenderBox {
 
     double computeIntrinsicAspectRatio() const;
 
+    void computeIntrinsicRatioInformation(FloatSize& intrinsicSize, double& intrinsicRatio) const override;
```

Comment:
Nit: Should have blank line after this method declaration.

Reply:
Changed